doc/user_guide/io.rst

```diff
@@ -75,6 +75,13 @@ Almost all file readers support accessing the data without reading it to memory
 analysing large files. To load a file without loading it to memory simply set
 ``lazy`` to ``True`` e.g.:
 
+The units of the navigation and signal axes can be converted automatically 
+during loading using the ``convert_units`` parameter. If `True`, the 
+``convert_to_units`` method of the ``axes_manager`` will be used for the conversion.
+If `None` (default), the default setting of the file reader will be used to 
```

Comment:
I think that this part of the documentation needs updating to take into account the latest changes.